### PR TITLE
Migrating over ops and server docs.

### PR DESCRIPTION
**Pull request by gcoonrod**

### CODE REVIEW

File: initializers/redis.js [hunk 1]
```diff
@@ -76,11 +76,11 @@ module.exports = class Redis extends ActionHero.Initializer {
      * @param  {string}  method         The method to call on the remote server.
      * @param  {Array}   args           The arguments to pass to `method`
      * @param  {string}  connectionId   (optional) Should this method only apply to a server which `connectionId` is connected to?
-     * @param  {Boolean}  waitForRespons (optional) Should we await a response from a remote server in the cluster?
+     * @param  {Boolean}  waitForResponse (optional) Should we await a response from a remote server in the cluster?
      * @return {Promise}                The return value from the remote server.
      */
-    api.redis.doCluster = async (method, args, connectionId, waitForRespons) => {
-      if (waitForRespons === undefined || waitForRespons === null) { waitForRespons = false }
+    api.redis.doCluster = async (method, args, connectionId, waitForResponse) => {
```
Comment (evantahler) [inline initializers/redis.js]: whoops!

File: tutorials/chat.md [hunk 2]
```diff
@@ -1,111 +0,0 @@
...
-*   `roomView`
-*   `say`
-
-The special verb for persistent connections `say` makes use of `api.chatRoom.broadcast` to tell a message to all other users in the room, IE: `say myRoom Hello World` from a socket client or `client.say("myRoom", 'Hello World")` for a websocket.
-
-Chat on multiple actionHero nodes relies on redis for both chat (pub/sub) and a key store defined by `api.config.redis`. Note that if you elect to use fakeredis, you will be using an in-memory redis server rather than a real redis process, which does not work to share data across nodes. The redis store and the key store don't need to be the same instance of redis, but they do need to be the same for all ActionHero servers you are running in parallel. This is how ActionHero scales the chat features.
-
-There is no limit to the number of rooms which can be created, but keep in mind that each room stores information in redis, and there load created for each connection.
-
-## Methods
```
Comment (evantahler) [inline tutorials/chat.md]: I agree the bulk of this page (`methods`) doesn't need to be here, but I think the intro and client-use parts are still important.

File: tutorials/websocket.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+Data is always returned as JSON objects to the webSocket client.
+
+An example web socket session might be the following:
+
+You can also inspect `client.state` (‘connected', ‘disconnected', etc). The websocket client will attempt to re-connect automatically.
+
+If you want to communicate with a websocket client outside of an action, you can call `connection.send(message)` on the server. In the client lib, the event message will be fired. So, `{`client.on('message, function(m){ ... })`}`. Be sure to add some descriptive content to the message you send from the sever (like perhaps `{`{"type": 'message type'}`}`) so you can route message types on the client.
+
+## Client Methods
```
Comment (evantahler) [inline tutorials/websocket.md]: We should probably move these to a *virtual* `websocket_client.jsdoc` file, and consider it a `class`. The only place these methods show up shouldn't be a tutorial :/

Comment (gcoonrod) [reply tutorials/websocket.md]: Do we have an examples folder or the like that can house jsdoc'd files?

Comment (evantahler) [reply tutorials/websocket.md]: I took care of it! https://github.com/actionhero/actionhero/commit/2fc428b99b7277501021a18dd2132c7952c0f7dc

https://docs.actionherojs.com/ActionHero.ActionHeroClient.html

**Pull request merged**